lib/clamp-device.in

```diff
@@ -190,11 +169,31 @@ if [ $KMDUMPLLVM == "1" ]; then
   cp $2.linked.bc ${KMDUMPDIR}/dump.linked.bc
 fi
 
+# Invoke HCC-specific opt passes
+if [ "$HCC_TILECHECK" == "ON" ]; then
+    $OPT -load $LIB/LLVMSelectAcceleratorCode@CMAKE_SHARED_LIBRARY_SUFFIX@ \
+        -select-accelerator-code -dce -globaldce < $2.linked.bc -o $2.selected.bc
+    $OPT -always-inline -tile-uniform < $2.selected.bc -o $2.selected.bc
+else
+    $OPT -load $LIB/LLVMSelectAcceleratorCode@CMAKE_SHARED_LIBRARY_SUFFIX@ \
+        -select-accelerator-code -dce -globaldce < $2.linked.bc -o $2.selected.bc
+    $OPT -always-inline < $2.selected.bc -o $2.selected.bc
```

Comment:
@AlexVlx could we place `-always-inline` right after `-globaldce` in the previous `opt` invocation? So we save one process.